paasta_tools/cli/cmds/rerun.py

```diff
@@ -0,0 +1,139 @@
+             "For example: --clusters norcal-prod,nova-prod"
+    ).completer = lazy_choices_completer(list_clusters)
+    rerun_parser.add_argument(
+        '-d', '--execution_date',
+        help="The date the job should be rerun for. Expected in the format %%Y-%%m-%%dT%%H:%%M:%%S .",
+        required=True,
+        type=type_execution_date
+    )
+    rerun_parser.set_defaults(command=paasta_rerun)
+
+
+def _get_cluster_instance(cluster_dot_instance_list):
+    """given a list of cluster.instance, returns
+    a nested dict of ['cluster']['instance'] = True"""
+    cluster_instance_dict = {}
+    for namespace in cluster_dot_instance_list:
+        cluster_in_pipeline, instance = namespace.split(SPACER, 1)
+        instance_dict = cluster_instance_dict.get(cluster_in_pipeline, {})
+        instance_dict[instance] = True
+        cluster_instance_dict[cluster_in_pipeline] = instance_dict
+    return cluster_instance_dict
+
+
+def paasta_rerun(args):
+    """Reruns a Chronos job.
+    :param args: argparse.Namespace obj created from sys.args by cli"""
+    service = figure_out_service_name(args)  # exit with an error if the service doesn't exist
+    execution_date = args.execution_date.strftime(EXECUTION_DATE_FORMAT)
+
+    all_clusters = list_clusters()
+    actual_deployments = get_actual_deployments(service)  # cluster.instance: sha
+    if actual_deployments:
+        deploy_pipeline = list(get_planned_deployments(service))  # cluster.instance
+        deployed_clusters = list_deployed_clusters(deploy_pipeline, actual_deployments)
+        deployed_cluster_instance = _get_cluster_instance(actual_deployments.keys())
+
+    if args.clusters is not None:
+        clusters = args.clusters.split(",")
+    else:
+        clusters = deployed_clusters
+
+    for cluster in clusters:
+        print "cluster: %s" % cluster
+
+        if cluster not in all_clusters:
+            print "  Warning: \"%s\" does not look like a valid cluster..." % cluster
+            continue
+        if cluster not in deployed_clusters:
+            print "  Warning: service \"%s\" has not been deployed to \"%s\" yet..." % (service, cluster)
+            continue
+        if not deployed_cluster_instance[cluster].get(args.instance, False):
+            print ("  Warning: instance \"%s\" is either invalid "
+                   "or has not been deployed to \"%s\" yet..." % (args.instance, cluster))
+            continue
+
+        rc, output = execute_chronos_rerun_on_remote_master(service,
+                                                            args.instance,
+                                                            cluster,
+                                                            verbose=args.verbose,
+                                                            execution_date=execution_date)
+        if rc == 0:
+            print PaastaColors.green('  success')
```

Comment:
can you make this 'successfully created job'? This might mislead users into thinking that this success is 'the job has run'